scripts/devenv-builder/configure-vm.sh

```diff
@@ -137,35 +174,59 @@ else
     rm -f ${OCC_LOC}
 fi
 
-if [ $BUILD_AND_INSTALL = true ] ; then
+if $BUILD_AND_RUN || $FORCE_FIREWALL; then
     # Run MicroShift Executable > Configuring MicroShift > Firewalld
     # https://github.com/openshift/microshift/blob/main/docs/howto_firewall.md#firewalld
-    sudo dnf install -y firewalld
-    sudo systemctl enable firewalld --now
-    sudo firewall-cmd --permanent --zone=trusted --add-source=10.42.0.0/16
-    sudo firewall-cmd --permanent --zone=trusted --add-source=169.254.169.1
-    sudo firewall-cmd --permanent --zone=public --add-port=80/tcp
-    sudo firewall-cmd --permanent --zone=public --add-port=443/tcp
-    sudo firewall-cmd --permanent --zone=public --add-port=5353/udp
-    sudo firewall-cmd --permanent --zone=public --add-port=30000-32767/tcp
-    sudo firewall-cmd --permanent --zone=public --add-port=30000-32767/udp
-    sudo firewall-cmd --permanent --zone=public --add-port=6443/tcp
-    sudo firewall-cmd --permanent --zone=public --add-service=mdns
-    sudo firewall-cmd --reload
+    title "Installing and configuring firewalld"
+    sudo dnf install -y firewalld >/dev/null
+    sudo systemctl enable firewalld --now >/dev/null
+    sudo firewall-cmd --permanent --zone=trusted --add-source=10.42.0.0/16 >/dev/null
+    sudo firewall-cmd --permanent --zone=trusted --add-source=169.254.169.1 >/dev/null
+    sudo firewall-cmd --permanent --zone=public --add-port=80/tcp >/dev/null
+    sudo firewall-cmd --permanent --zone=public --add-port=443/tcp >/dev/null
+    sudo firewall-cmd --permanent --zone=public --add-port=5353/udp >/dev/null
+    sudo firewall-cmd --permanent --zone=public --add-port=30000-32767/tcp >/dev/null
+    sudo firewall-cmd --permanent --zone=public --add-port=30000-32767/udp >/dev/null
+    sudo firewall-cmd --permanent --zone=public --add-port=6443/tcp >/dev/null
+    sudo firewall-cmd --permanent --zone=public --add-service=mdns >/dev/null
+    sudo firewall-cmd --reload >/dev/null
+fi
 
+if $BUILD_AND_RUN; then
     # Run MicroShift Executable > Configuring MicroShift
     # https://github.com/openshift/microshift/blob/main/docs/devenv_setup.md#configuring-microshift
-    sudo systemctl enable crio
-    sudo systemctl start microshift
+    title "Starting CRI-O"
+    sudo systemctl enable crio >/dev/null
+
+    title "Enabling and starting MicroShift"
+    sudo systemctl enable microshift --now >/dev/null
+
+    sudo bash -c "while [ ! -e /var/lib/microshift/resources/kubeadmin/kubeconfig ]; do sleep 5s; done"
+
+    title "Copying kubeconfig to ~/.kube/config"
+    mkdir -p ~/.kube
+    sudo cat /var/lib/microshift/resources/kubeadmin/kubeconfig | tee ~/.kube/config >/dev/null
```

Comment:
I'm not sure this is necessary here. The idea is to verify MicroShift can be started and clean it up, so that it's possible to run standalone exe in the devenv. 
So, I would remove this and fix the `watch oc get pod -A` command to use the original kubeconfig file.

Reply:
removed